src/pages/ReimbursementAccount/VerifiedBankAccountFlowEntryPoint.tsx

```diff
@@ -219,8 +219,11 @@ function VerifiedBankAccountFlowEntryPoint({
                     )}
                     {shouldShowContinueSetupButton === true ? (
                         <OfflineWithFeedback
-                            errors={getLatestError(errors)}
+                            errors={
+                                reimbursementAccount?.maxAttemptsReached ? getMicroSecondOnyxErrorWithTranslationKey('connectBankAccountStep.maxAttemptsReached') : getLatestError(errors)
+                            }
                             shouldShowErrorMessages
+                            canDismissError={!reimbursementAccount?.maxAttemptsReached}
```

Comment:
@Expensify/design what do you think about this change? This is making the error persistent and not allowing you to dismiss it. I'd argue that you can dismiss the error after you've seen it, then that is on you because you saw it and dismissed it. The error will still be presented when they try to setup the bank account again on a different screen.

Reply:
Hmm I don't feel too strongly. Normally we allow users to dismiss the error, although in this case, if you dismiss the error, the buttons would still be disabled which might feel weird right?

Reply:
So I think I would keep it there as not dismissable. What makes it so that you can log in again though?

Reply:
This is for adding a bank account. So in this case, we're showing an error because they've already failed too many times to add a bank account, and their next step is to reach out to Concierge.

If they dismiss the error, they can go through the flow again but then they'll hit the same error page. I think in that case, making it non dismissable is fine since the outcome is the same, right?